.github/scripts/hermetic_library_generation.sh

```diff
@@ -54,22 +47,18 @@ shift
 done
 
 if [ -z "${target_branch}" ]; then
-  echo "missing required argument --target_branch"
+  echo "Missing required argument --target_branch"
   exit 1
 fi
 
 if [ -z "${current_branch}" ]; then
-  echo "missing required argument --current_branch"
+  echo "Missing required argument --current_branch"
   exit 1
 fi
 
 if [ -z "${generation_config}" ]; then
   generation_config=generation_config.yaml
-  echo "Use default generation config: ${generation_config}"
```

Comment:
Can we keep the image tag as a parameter? In long term, we do want to pass it in and remove the generator version from the generation_config.yaml.

Reply:
Sorry I missed this comment. I'll create a follow up.